src/Raven.Embedded/RavenServerRunner.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+namespace Raven.Embedded
+{
+    internal class RavenServerRunner
+    {
+        private static readonly string RavenDbServerPath = Path.Combine(AppContext.BaseDirectory, "RavenDBServer/Raven.Server.dll");
+
+        public static Process Run(ServerOptions options)
+        {
+            if (File.Exists(RavenDbServerPath) == false)
+            {
+                throw new FileNotFoundException("Server file was not found", RavenDbServerPath);
+            }
+
+            using (var currentProcess = Process.GetCurrentProcess())
+            {
+                options.CommandLineArgs.Add("--Embedded.ParentProcessId=" + currentProcess.Id);
+            }
+
+            if(string.IsNullOrEmpty(options.DataDir) == false)
+                options.CommandLineArgs.Add($"--DataDir={options.DataDir}");
+
+            var argumentsString = string.Join(" ", new[] { $"--fx-version { options.FrameworkVersion} ", RavenDbServerPath }
```

Comment:
if FxVersion is not `null` then you should not add the `--fx-version` parameter

Reply:
?

Reply:
Too many `not's`. Oren fixed that.